pyvista/plotting/camera.py

```diff
@@ -340,3 +340,81 @@ def view_frustum(self, aspect=1.0):
 
         frustum = pyvista.wrap(frustum_source.GetOutput())
         return frustum
+
+    @property
+    def roll(self):
+        """Rotate the camera about the direction of projection.
+
+        This will spin the camera about its axis.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> pl = pyvista.Plotter()
+        >>> pl.camera.roll
+        -120.00000000000001
+        """
+        return self.GetRoll()
+
+    @roll.setter
+    def roll(self, angle):
+        """Set the rotate of the camera about the direction of projection.
+
+        This will spin the camera about its axis.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> pl = pyvista.Plotter()
+        >>> pl.camera.roll = 45.0
+        """
+        self.SetRoll(angle)
+
+    def vertical_rotate(self, angle):
```

Comment:
Can we make this a setter/getter like `roll`?

Reply:
Yes. In that case, we should calculate `vertical_rotate` in pyvista. Please wait while I make it.